src/app/api/senior/route.ts

```diff
@@ -48,7 +49,7 @@ export const POST = withSessionAndRole(
       }
       const baseFolder = "1MVyWBeKCd1erNe9gkwBf7yz3wGa40g9a"; // TODO: make env variable
       const fileMetadata = {
-        name: [`${body.name}-${randomUUID()}`],
+        name: [`${seniorBody.firstname}${seniorBody.lastname}-${randomUUID()}`],
```

Comment:
This might be a bit hard to read on Google Drive. Should we consider `firstname_lastname` instead?